### ISSUE TITLE
WoL Bartz and WoL Lenna latents ?

### ISSUE THREAD

**Issue by Wisdomb33r** [aEnigmatic/ffbe#16]
Hi aEnigma,

Since a few weeks I'm going nut about WoL Bartz and WoL Lenna latent abilities. I cannot find in your unit_latent_skills.json anything related to these units.

205001105, 205001106 and 205001107 (the ID's for Bartz) are not found in this file. But 100011705 (ID for the 5* PG Lasswell) is.

I also tried to start from the skill 800028 (ID for his passive latent), but could also not find anything.

Do you have any clue how the game can properly propose the latent skills for these characters IG, while your dump does not reflect these unit/skill links ? Apparently all other characters having latent abilities are okay.

Thanks in advance for your help.

**Comment by aEnigmatic** [comment]
Hey, I'll take a look tomorrow if I find the time

**Comment by aEnigmatic** [comment]
It seems that - for whatever reason - those are not in the game files but are sent server side:
```
                {
                    "SeasonEventGroupId": "9090501",
                    "TargetType1": "3",
                    "TargetId1": "205001107",
                    "TargetType2": "3",
                    "TargetId2": "500",
                    "TargetFlg2": "1",
                },
```

Are they supposed to be permanent?

**Comment by aEnigmatic** [comment]
Fixed with cc7577eb.
